.github/workflows/integration_test.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       builder-runner-label: X64
 
   integration-tests-arm:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@feat/private_endpoint
```

Comment:
Recent merges in craft-cli makes the testing for operator-workflow fail. Use this until that gets resolved.

Reply:
See https://github.com/canonical/rockcraft/issues/621

Reply:
Should be fixed now